src/utils/repo/browser.js

```diff
@@ -21,6 +16,14 @@ exports.repoExists = function repoExists (repoPath, cb) {
       const table = store.table(repoPath)
       return table
         .count((cnt) => cb(null, cnt > 0))
-        .catch(cb)
-    }).catch(cb)
+        .catch(e => cb(null, false))
+    }).catch(e => cb(null, false))
+}
+
+exports.defaultRepo = function (type) {
+  return 'ipfs'
+}
+
+exports.checkForRunningApi = function (path) {
+  return null
```

Comment:
what is happening here?

Reply:
its just browser version, node version checks the file system for the api file in the browser we don't have fs :)